test/src/unit-cppapi-var-offsets.cc

```diff
@@ -60,22 +60,43 @@ void create_sparse_array(const std::string& array_name) {
 }
 
 void write_sparse_array(
+    Context ctx,
     const std::string& array_name,
     std::vector<int32_t>& data,
-    std::vector<uint64_t>& data_offsets) {
-  // Write some variable-sized data in the array
+    std::vector<uint64_t>& data_offsets,
+    tiledb_layout_t layout) {
   std::vector<int64_t> d1 = {1, 2, 3, 4};
   std::vector<int64_t> d2 = {2, 1, 3, 4};
 
-  Context ctx;
   Array array(ctx, array_name, TILEDB_WRITE);
   Query query(ctx, array, TILEDB_WRITE);
-  query.set_layout(TILEDB_UNORDERED)
-      .set_buffer("d1", d1)
-      .set_buffer("d2", d2)
-      .set_buffer("attr", data_offsets, data);
+  query.set_layout(layout).set_buffer("d1", d1).set_buffer("d2", d2).set_buffer(
+      "attr", data_offsets, data);
```

Comment:
It may be beneficial to format this in a similar manner to how it was done before for better readability.
 i.e.
`query.set_layout(layout)
           .set_buffer("d1", d1)
           .set_buffer("d2", d2)
           .set_buffer("attr", data_offsets, data);`